src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9407 @@
+    m_actionTemplates[ScriptAction::SKIRMISH_PERFORM_COMMANDBUTTON_ON_MOST_VALUABLE_OBJECT].m_uiStrings[4] =
+        " (true = all valid sources, false = first valid source).";
+
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_internalName =
+        "SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_uiName =
+        "Skirmish_/ Delay a sequential script until the specified command ability"
+        " is ready - all.";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_numParameters = 3;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_parameters[0] = Parameter::SIDE;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_parameters[1] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_parameters[2] =
+        Parameter::UNK_ABILITY_RELATED;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_numUiStrings = 4;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_uiStrings[1] = " 's ";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_uiStrings[2] = " all wait until ";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_ALL].m_uiStrings[3] = " is ready.";
+
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_internalName =
+        "SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_uiName =
+        "Skirmish_/ Delay a sequential script until the specified command ability"
+        " is ready - partial.";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_numParameters = 3;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_parameters[0] = Parameter::SIDE;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_parameters[1] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_parameters[2] =
+        Parameter::UNK_ABILITY_RELATED;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_numUiStrings = 4;
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_uiStrings[1] = " 's ";
+    m_actionTemplates[ScriptAction::SKIRMISH_WAIT_FOR_COMMANDBUTTON_AVAILABLE_PARTIAL].m_uiStrings[2] =
+        " wait until at least one member is ";
```

Comment:
This sentence seems to make no sense grammar wise.

Reply:
This is correct.